### ISSUE TITLE
How to spawn objects and change conents in the scene.

### ISSUE THREAD

**Issue by a2a4** [isl-org/spear#275]
#### What platform am I on? What are my system specifications? Do I meet the minimum specifications for the Unreal Engine?
Platform: Ubuntu 20.04

#### What commit of the `spear` repository am I on? Am I synced to a specific tag, or am I synced to the head of the `main` branch?
Branch: main

#### What am I attempting to do, what do I expect to happen, and what actually happened?
I'm attempting to create my own scene in the simulation and spawn objects in the scene and create a scene in the environment.

Can you please suggest me how do I go about this? I've searched through the contents of the documentation. But it is not clear.

**Comment by mikeroberts3000** [comment]
Hi. We don't currently have good support for spawning custom objects. The spawning behavior in SPEAR is painfully hard-coded right now. But we'll be improving things significantly in our next release that we'll be shipping in a few weeks. Thanks for trying out SPEAR, and stay tuned for our next release!

**Comment by a2a4** [comment]
Hello @mikeroberts3000. Thanks for your prompt reply. Looking forward to the further releases of spear. 

Currently I've seen few scenes, in the spear release. debug_0000, debug1_0000, apartment_0000 and warehouse_0000. I extracted the SpearSim-Linux.pak and warehouse_0000-v0.4.0-Linux.pak through the ./UnrealPak /source/dir/ -Extract /destination/dir/ command to make changes to the warehouse scene for the way we desired. I was able to open the unreal levels of debug_0000, debug1_0000, apartment_0000 but not for warehouse_0000 level. There were no .umap or .uexp extension files in the extracted version of warehouse_0000-v0.4.0-Linux.pak.

Very grateful if we can receive your input on this. Thanks and cheers.
